app/Http/Controllers/Admin/AdminAuthController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace HotelBooking\Http\Controllers\Admin;
+
+use Auth;
+use Illuminate\Http\Request;
+use HotelBooking\Http\Requests;
+use HotelBooking\Http\Controllers\Controller;
+use HotelBooking\Http\Requests\Admin\AdminLoginRequest;
+
+class AdminAuthController extends Controller
+{
+
+    public function index(){
+      return "View index admin";
+    }
+
+    public function getLoginAdmin()
```

Comment:
This class is put in `Admin` directory, so that you don't need to add `Admin` as prefix or subfix. Action name should be `getLogin()`